### PR TITLE
Remove `ConsensusCreated` event

### LINKED ISSUE #155: Extinguish `ConsensusCreated` emitted by `Authority`
## 📚 Context

As soon as one deploys an `Authority` contract, it emits an event called [`ConsensusCreated`](https://github.com/cartesi/rollups/blob/main/onchain/rollups/contracts/consensus/authority/Authority.sol#L33-L37) which contains the following parameters:

* `owner`: the address that has ownership over the authority contract
* `inputBox`: the address of the [`IInputBox`](https://github.com/cartesi/rollups/blob/main/onchain/rollups/contracts/inputs/IInputBox.sol) that the authority will listen to `InputAdded` events

We argue that the `ConsensusCreated` event can be extinguished by showing that each parameter can already be retrieved in another way, or doesn't capture well how the Authority works in practice.

* `owner`: this information is already emitted through [`OwnershipTransferred`](https://docs.openzeppelin.com/contracts/2.x/api/ownership#Ownable-OwnershipTransferred-address-address-) events (see OpenZeppelin's [`Ownable`](https://docs.openzeppelin.com/contracts/2.x/api/ownership#Ownable) contract)
* `inputBox`: this assumes that the authority will listen to exactly one input box forever. This does not take multiple input boxes into consideration. This will be the case whenever (and if) a new version of the input box contract is released and deployed. Furthermore, it makes more sense for this parameter to be application-specific, and not consensus-specific. For example, an input box could allow bigger inputs because the off-chain machine running the DApp back-end allows.

## ✔️ Solution

We may remove the `ConsensusCreated` event from the `Authority` contract.
We need to make sure this is not a breaking change for the off-chain.

As a result, the `IInputBox` argument passed to the constructor of the `Authority` contract can be removed.
This change will have cascading effects in #64, #99, and #85.

## 📈 Subtasks

- [ ] Remove the event
- [ ] Update the on-chain tests
- [ ] Verify that the off-chain runs fine

### PR DESCRIPTION
**Pull request by guidanoli**
Closes #155

### CODE REVIEW

**Comment by pedroargento** [comment]
We need to adjust the factories to remove InputBox as a constructor parameter.

**Comment by guidanoli** [comment]
> We need to adjust the factories to remove InputBox as a constructor parameter.

Yes, this is foreseen by #155.
Once this PR is merged, we'll need to do rebase on these branches. :-)

File: onchain/rollups/contracts/consensus/authority/Authority.sol [hunk 1]
```diff
@@ -30,12 +30,6 @@ contract Authority is AbstractConsensus, Ownable {
     /// @dev See the `getHistory` and `setHistory` functions.
     IHistory internal history;
 
-    /// @notice The `Authority` contract was created.
-    /// @param owner The address that initially owns the `Authority` contract
-    /// @param inputBox The input box from which the authority fetches new inputs
-    /// @dev MUST be triggered during construction.
-    event ConsensusCreated(address owner, IInputBox inputBox);
```
Comment (ZzzzHui) [inline onchain/rollups/contracts/consensus/authority/Authority.sol:37]: you can also remove the import of `IInputBox`. Line 20

Comment (guidanoli) [reply onchain/rollups/contracts/consensus/authority/Authority.sol:37]: Hey, you're right! Thanks for spotting this. :-)

**Comment by guidanoli** [comment]
Rebased and remove line 20 in `Authority.sol` (thanks, @ZzzzHui!).

**Review by ZzzzHui — APPROVED** [review]

**Review by pedroargento — APPROVED** [review]

**Pull request merged**